pkg/operator/ceph/cluster/controller.go

```diff
@@ -550,6 +564,30 @@ func (c *ClusterController) onUpdate(oldObj, newObj interface{}) {
 		}
 		return
 	}
+
+	// Display success after upgrade
+	if versionChanged {
```

Comment:
How about if we always print a message such as "all ceph daemons are running version x"? Then we will know at the end of every orchestration that the versions are matching.

Reply:
Isn't what the `else` does anyway?